dashboard/bin/scheduled_ops_emails

```diff
@@ -12,7 +12,7 @@ def main
         facilitators = Workshop.find(workshop[:id]).facilitators
         [teachers, drop_ins, facilitators].each do |recipient_list|
           recipient_list.each do |recipient|
-            if workshop.segments.order(:start).last[:start].to_date == Date.today
+            if workshop.first_segment[:start].to_date == Date.today
```

Comment:
while [:start] will work .start is more idiomatic.. for one thing if you have a typo it'll tell you NoMethodError instead of returning nil